src/components/Md5Form.test.js

```diff
@@ -0,0 +1,28 @@
+import { render, screen } from '@testing-library/react';
+import { getMd5 } from '../providers/md5Provider';
+import userEvent from '@testing-library/user-event';
+import Md5Form from './Md5Form';
+
+jest.spyOn(window, 'fetch');
+
+describe('<Md5Form>', () => {
+  it('check if input exists', () => {
+    render(<Md5Form getMd5={getMd5} />);
+
+    const input = screen.getByLabelText('');
+
+    expect(input).toBeInTheDocument();
+  });
+
+  //   Mateusz, tutaj coś pojawia mi się problem, bo kompletnie nie odczytuje mi tekstu wpisanego w input. W efekcie otrzymuję pusty string, zamiast tego co wpisałem w userEvent. Bez tego nie mogę ruszyć z pozostałymi rzeczami. Z góry dzięki za pomoc!
+
+  it('check if input value updates span', () => {
```

Comment:
Na tym etapie powinieneś zaimplementować mock dla `fetch`, aby nie przyśpieszyć działanie całego kodu.

Obecnie po wpisaniu danych do inputa czekasz na odpowiedź z API. To powoduje, że nie ma tych danych od razu, kiedy ty wyszukujesz ten element. Pamiętaj też, że czasem trzeba skorzystać z `await`, aby przegenerowanie zostało wykonane. 

Coś na zasadzie: `const userLoggedIn = await screen.findByText(`Jesteś zalogowany jako: ${login}`);`